gst/mqtt/mqttsink.c

```diff
@@ -268,6 +273,15 @@ gst_mqtt_sink_class_init (GstMqttSinkClass * klass)
           -1, G_MAXINT32, DEFAULT_NUM_BUFFERS,
           G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (gobject_class, PROP_MQTT_QOS,
+      g_param_spec_uint ("mqtt-qos", "mqtt QoS level",
```

Comment:
```MQTTAsync_subscribe () ``` and ```MQTTAsync_send ( )``` take an integer to set the QoS mode. https://www.eclipse.org/paho/files/mqttdoc/MQTTAsync/html/_m_q_t_t_async_8h.html#a08d18ece91c1b011011354570d8ac1ab

Reply:
got it, it's updated!